src/test/java/com/google/javascript/clutz/type_renaming_with_emit_platform_externs.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Test that during the conversion of platform externs, the types below
+ * aren't renamed
+ * @externs
```

Comment:
I think it will be cleaner if this file was "intentionally empty", and the ".externs.js" contains the current contents.